src/main/java/gregtech/loaders/recipe/ComponentRecipes.java

```diff
@@ -379,5 +383,17 @@ public static void register() {
                     .outputs(MetaItems.ELECTRIC_PUMP_IV.getStackForm())
                     .duration(100).EUt(7680).buildAndRegister();
         }
+
+        Material[] circuitTiers = new Material[] {MarkerMaterials.Tier.Basic, MarkerMaterials.Tier.Good, MarkerMaterials.Tier.Advanced, MarkerMaterials.Tier.Extreme};
```

Comment:
This should go up to IV tier, `MarkerMaterials.Tier.Elite` to match all the other component recipes in GTCE being craftable up to IV.